pkgs/development/python-modules/pypng/default.nix

```
@@ -8,14 +8,14 @@
 
 buildPythonPackage rec {
   pname = "pypng";
-  version = "0.20220715.0";
+  version = "0.20231004.0";
   pyproject = true;
 
   src = fetchFromGitLab {
     owner = "drj11";
     repo = "pypng";
-    rev = "refs/tags/${pname}-${version}";
-    hash = "sha256-tTnsGCAmHexDWm/T5xpHpcBaQcBEqMfTFaoOAeC+pDs=";
+    tag = "${pname}-${version}";
```

```suggestion
    tag = "pypng-${version}";
```
technically we are wanting to stop doing this 🙈